src/index.js

```diff
@@ -271,7 +270,7 @@ class DependencyGraph {
     // After we process a file change we record any errors which will also be
     // reported via the next request. On the next file change, we'll see that
     // we are in an error state and we should decide to do a full rebuild.
-    this._loading = this._loading.finally(() => {
+    this._loading = this._loading.then(() => {
```

Comment:
I think this change is incorrect. `finally` runs both on the error and the success state, so with your change we'll get stuck while file watching if there is an error.